packages/github/src/github.ts

```diff
@@ -61,14 +61,19 @@ export class GitHub extends Octokit {
     const token = args[0]
     const options = {...args[1]} // Shallow clone - don't mutate the object provided by the caller
 
+    // Base URL - GHES or Dotcom
```

Comment:
I'll look before merging whether any unit test can be added. Might be able to verify the expected baseUrl is set

Reply:
unfortunately doesnt appear to be a way to get the options after it's constructed